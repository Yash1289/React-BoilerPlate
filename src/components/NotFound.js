const React = require("react");

const NotFound = () => (
    <div>
        <p>404!</p>
    </div>
)

export default NotFound