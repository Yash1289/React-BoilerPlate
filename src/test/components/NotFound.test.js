import { TestScheduler } from "jest"
import React from "react"
import { shallow } from "enzyme"
import NotFound from "../../components/NotFound"


test("Should render header component ", () => {
    const wrapper = shallow(<NotFound />)
    expect(wrapper).toMatchSnapshot(); 
})