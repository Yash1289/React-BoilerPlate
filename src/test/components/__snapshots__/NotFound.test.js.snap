// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header component  1`] = `
<div>
  <p>
    404!
  </p>
</div>
`;
